### ISSUE TITLE
Single Player Campaign Crashes

### ISSUE THREAD

**Issue by Hicks613** [sd805/l4d2vr#19]
New L4D2 folder, so no mods or subscriptions are present.
I dumped the zip file contents in my L4d2 root.  the Dll's are a long side the l4d2 exe.

Steam VR and SteamVR beta.
Theater mode disabled and disabled.
Graphic options match your screenshot
DSR is diabled, but I have tried multiple settings in nvidia.
THe game's menu is fine but starting any game cues the audio, there is disk activity, then crash to desktop.

AMD 5950x
16gb ram
rtx 3080 10gb

**Comment by Hicks613** [comment]
Update:
Played with full screen, windows and windowed borderless.  Turned PAGE POOL MEMORY to high and then low and got further.

The game launches.  F6 and A on my controllerse to nothing to initiate VR.  I can look around with mouse, but can't click to shoot.
SteamVR is stuck on my 360 wallpaper sometimes on Next UP... and sometimes saiting Waiting with the left 4 dead title in the window.

I've tried using virtual desktop and Oculus link.

**Comment by Richleth** [comment]
Same issue, F6 and A do nothing

**Comment by Hicks613** [comment]
> Same issue, F6 and A do nothing
VR aside, does your game work?  I mean does it move about on the flatscreen and can you actually shoot using motion controls?
Or is the only thing that works just keyboard and mouse (or gamepad if you enabled it).

**Comment by Hicks613** [comment]
Just a bit of a dump of things that I did to fix this.
1. MSI afterburner seems to have installed Riva statistics tuner.  This was removed
2. Windows defender was set to ignore my steam folder (and all subfolders).  Oculus as well.  Before this, I don't think Defender was allowing windows to read the DLL files that came with this mod as log files were not being generated.
3. I'm still having some trouble with Virtual Desktop - I think this is the way I'm viewing my desktop and starting the campaign there using my controllers as a mouse.  But Link/Airlink continues to work.
